### PR TITLE
Fixes to previous PRs

### PR DESCRIPTION
**Pull request by Symmachus**

### CODE REVIEW

**Comment by loup99** [comment]
Could you resolve the conflicts with master or do you intend on continuing work?

**Comment by Symmachus** [comment]
> Could you resolve the conflicts with master or do you intend on continuing work?

I resolved conflicts

**Comment by IhateTrains** [comment]
I think you've changed the encoding in WTWSMS/common/cultures/00_cultures.txt. The rest of anglicization removal seems fine, good job 👍

File: WTWSMS/common/landed_titles/landed_titles.txt [hunk 1]
```diff
@@ -51726,7 +51726,7 @@ b_theodosius = {
 	culture = romano_hispanic
 	religion = nicene
 }
-b_justinian = {
+b_Iustinian = {
```
Comment (loup99) [inline WTWSMS/common/landed_titles/landed_titles.txt]: Since it is used elsewhere and isn't a localisation line, it is better to keep this as it is an change its localisation instead.

File: WTWSMS/history/characters/parthian.txt [hunk 2]
```diff
@@ -510,7 +510,6 @@
 	religion="zoroastrian"
 	culture="parthian"
 	father=214055
-	trait=mazdeist
```
Comment (loup99) [inline WTWSMS/history/characters/parthian.txt]: Why did you remove this?

File: WTWSMS/history/characters/parthian.txt [hunk 3]
```diff
@@ -520,12 +519,11 @@
 }
 
 214057 = {
-	name="Valagash" # Vologases II
+	name="Valakhsh" # Vologases II
 	dynasty=100685
 	religion="zoroastrian"
 	culture="parthian"
 	father=214055
-	trait=mazdeist
```
Comment (loup99) [inline WTWSMS/history/characters/parthian.txt]: Same as above.

File: WTWSMS/history/characters/parthian.txt [hunk 4]
```diff
@@ -540,7 +538,6 @@
 	religion="zoroastrian"
 	culture="parthian"
 	father=159547
-	trait=mazdeist
```
Comment (loup99) [inline WTWSMS/history/characters/parthian.txt]: Same too.

File: WTWSMS/history/characters/parthian.txt [hunk 5]
```diff
@@ -577,34 +574,6 @@
...
-	father=159550
-	165.1.1={
-		birth=yes
-	}
-	217.1.1={
-		death=yes
-	}
-}
-
-159579 = {
```
Comment (loup99) [inline WTWSMS/history/characters/parthian.txt]: Is this character completely removed?

**Comment by Symmachus** [comment]
> I think you've changed the encoding in WTWSMS/common/cultures/00_cultures.txt. The rest of anglicization removal seems fine, good job

I don't think so. The fill is still encoded in ANSI.

**Comment by loup99** [comment]
> > I think you've changed the encoding in WTWSMS/common/cultures/00_cultures.txt. The rest of anglicization removal seems fine, good job
> 
> I don't think so. The fill is still encoded in ANSI.

There is some problem, I would advice that you redownload that file through master and add your changes on top of that.

File: WTWSMS/history/characters/parthian.txt (hunk 2 above)
Comment (Symmachus) [reply WTWSMS/history/characters/parthian.txt]: When first time i worked on the Arsacids, i wanted to add zoroastrian school trait to them. But then, because i didn't found anything that clearly says about belonging of the Arsacids to any school, i changed my mind and removed traits that i previously added. Thereat i forgot to removed school traits from these characters, so i done it now. I think we could made them Zurvanists, as an oppose to the Mazdeist Sassanians.

File: WTWSMS/history/characters/parthian.txt (hunk 5 above)
Comment (Symmachus) [reply WTWSMS/history/characters/parthian.txt]: Yes, i accidentaly added him twice.

File: WTWSMS/history/characters/parthian.txt (hunk 2 above)
Comment (loup99) [reply WTWSMS/history/characters/parthian.txt]: Ok, the player and AI gets a choice when starting their game.

File: WTWSMS/common/landed_titles/landed_titles.txt (hunk 1 above)
Comment (Symmachus) [reply WTWSMS/common/landed_titles/landed_titles.txt]: I will fix it.

**Comment by loup99** [comment]
Tell us when the culture file issue can be resolved, so that we can continue with reviewing and then merging.

**Comment by Symmachus** [comment]
Now i think that encoding of the culture files has been finally fixed.

**Comment by loup99** [comment]
> Now i think that encoding of the culture files has been finally fixed.

Yes, the file is no longer mentioned as modified!
